api/create_account_and_login_test.go

```diff
@@ -43,7 +44,9 @@ func TestCreateAccountAndLogin(t *testing.T) {
 	var request requests.CreateAccount
 	err := json.Unmarshal([]byte(requestJSON), &request)
 	require.NoError(t, err)
-	statusBackend := NewGethStatusBackend()
+	logger, err := zap.NewDevelopment()
+	require.NoError(t, err)
```

Comment:
do you think we could extract this to a function into some `status-go-testing-helpers` package? (with a better short name ofc)
Then we would have a single place of modifying the logger used in tests.

Reply:
It already exists, I forgot. I've changed it to `tt.MustCreateTestLogger`.